utilities/checkpoint/checkpoint.cc

```diff
@@ -211,15 +211,16 @@ Status CheckpointImpl::CreateCheckpoint(const std::string& checkpoint_dir) {
     std::vector<std::string> subchildren;
     db_->GetEnv()->GetChildren(full_private_path, &subchildren);
     for (auto& subchild : subchildren) {
-      Status s1 = db_->GetEnv()->DeleteFile(full_private_path + subchild);
+      std::string subchild_path = full_private_path + "/" + subchild;
+      Status s1 = db_->GetEnv()->DeleteFile(subchild_path);
       if (s1.ok()) {
-        Log(db_->GetOptions().info_log, "Deleted %s",
-            (full_private_path + subchild).c_str());
+        Log(db_->GetOptions().info_log, "Delete file %s -- %s",
```

Comment:
can you s/Delete/Deleted/ ? I think past tense is clearer for completed operations.